draft-pauly-taps-arch.md

```diff
@@ -213,9 +213,9 @@ Beyond the basic objects, there are several high-level groups of actions that an
  +-->| Preconnection |----------->| Connection |---------------> Closed
  |   +---------------+     :      +------------+ Connection:
  |                         :      ^   ^    |     Finished  :
- +-- Local Endpoint        :      |   |    |               :
+ +-- LocalEndpoint         :      |   |    |               :
```

Comment:
was this an overcorrection to CamelCase? Spaces (we should use "Local Endpoint", "Remote Endpoint" in the architecture draft; whether we camelcase in the API or just call things Local and Remote is a separate discussion

Reply:
Agreed that this should be "Local Endpoint" and "Remote Endpoint"

Reply:
Fixed.